_data/authors.yml

```diff
@@ -34,3 +34,10 @@ lucia_payo:
   bio: ""
   avatar: authors/lucia_payo.gif
   twitter: LuciaP_86
+
+henning_gross:
+  name: Henning Groß
+  web: http://www.upday.com
+  email: igaffai@gmail.com
+  bio: "Henning leads the technical team at upday as director engineering. He is still passionate about code and tries to work as operational as possible for someone in a management position (which can be painful for the team at times)."
```

Comment:
could you please use the comments that Maciej added around my author block and use bio and bio_long appropriately. Please run the blog on your local machine to see how the author part looks in the bottom of the page.